### PR TITLE
bcfg2 server sends a complete SELinux context, but we only need the t…

### PR DESCRIPTION
**Pull request by gordonmessmer**
bcfg2 server sends a complete SELinux context, but we only need the type.  Treat this the same as selinux.matchpathcon.

### CODE REVIEW

**Comment by stpierre** [comment]
Just a note: Travis-CI can't test the SELinux bits, so we'll need to be sure to run the unit tests manually. I'll try to take a look at this tomorrow.

**Comment by solj** [comment]
@stpierre If you can give me a hint on how to run the tests manually, I can take care of it.

**Comment by stpierre** [comment]
You should just be able to run:

    nosetests testsuite

You'll need to have all of the various requirements, including python-selinux, installed. python-selinux *can't* be installed into a virtualenv, so if you're not running CentOS/Fedora/RHEL on your workstation, or don't want to install it locally, then this might be a good case for a Docker container.

**Comment by gordonmessmer** [comment]
Any progress on this?

**Comment by solj** [comment]
I'm seeing no issues with this, however, I am seeing a ton of other errors so perhaps I'm missing it.

**Comment by gordonmessmer** [comment]
Let me rephrase that:  Is there anything I can do to help get these two SELinux patches merged?

**Comment by gordonmessmer** [comment]
I'll note that mock runs the test suite with all prereqs in a clean environment.  Would it help if I provided the build logs with the test suite results?

These patches are two months old.  My coworkers see bcfg2 as a lost cause, and the project is being called "near abandonware" at conferences.  I'm offering to help, but while it seems pretty clear that more help is needed, it doesn't seem like help is being sought.

**Comment by solj** [comment]
I'm good with the patches if the mock tests run cleanly for you. :+1:

**Comment by gordonmessmer** [comment]
Yes, the test suite runs successfully in mock, in both CentOS 6 and CentOS 7 build roots.

**Comment by solj** [comment]
Merged, thanks.
